package.json

```diff
@@ -33,7 +33,9 @@
     "build:esm": "tsc --declaration --module esnext --outDir dist/esm",
     "preupdate-plugins": "tsc --build tsconfig.update.json",
     "update-plugins": "node dist/cjs/util/update-plugins.js",
-    "postupdate-plugins": "npm run clean"
+    "postupdate-plugins": "npm run clean",
+    "lint": "eslint '**/*.ts' && npx prettier --check '**/*.ts'",
```

Comment:
We should add `lint-staged` and `husky` to enforce this linting on commit / push. Also I think in general we usually just rely on eslint to handle the prettier cases with what is set up in the `eslintrc` -- using both here can sometimes lead to weird conflicts where one overwrites the other (https://www.robinwieruch.de/prettier-eslint/)